regexes/device/mobiles.yml

```
@@ -1512,9 +1529,22 @@ Avvio:
 
 # BGH
 BGH:
-  regex: '(?<!Positivo )BGH ([^/;]+) Build'
+  regex: '(?!Positivo )BGH ([^/;]+)(?: Build|\))'
   device: 'smartphone'
-  model: '$1'
+  models:
+    - regex: 'Y([17]00|2[01]0|1010)'
+      device: 'tablet'
+      model: 'Y$1'
+    - regex: 'BGH (\+Simple|Mini)'
+      device: 'tablet'
+      model: '$1'
+    - regex: 'Y710 KIDS'
+      device: 'tablet'
+      model: 'Y710 Kids'
+    - regex: 'JOY X2'
+      model: 'Joy X2'
+    - regex: '(?!Positivo )BGH ([^/;]+)(?: Build|\))'
```

No need to check for not `Positivo` here again, as it won't check that regex if it is there...
```suggestion
    - regex: 'BGH ([^/;\)]+)(?: Build|\))'
```